scripts/exrrfs_analysis_gsi.sh

```diff
@@ -1029,7 +1029,7 @@ EOF
 #
 #-----------------------------------------------------------------------
 #
-if [ ${DO_DACOLD} = "FALSE" ]; then
+if [ ${DO_DACOLD} = "FALSE" ] && [ ${BKTYPE} -eq 1 ]; then
```

Comment:
Should the variables be quoted (e.g., `"${DO_DACOLD}"`)? I also see both instances of `[...] && [...]` and `[[...&&...]]` throughout the code. Not sure which is better/preferred/safer if any?